utils/utils_test.go

```diff
@@ -85,11 +85,6 @@ func TestSanitizeHexString(t *testing.T) {
 			hash:           "1234",
 			expectedResult: "1234",
 		},
-		{
```

Comment:
No longer needed since `strings.TrimPrefix` has no such requirements. This behavior was always a little odd, since `SanitizeHexString` isn't intended to be a verification function.